src/datastore/pg-store.ts

```diff
@@ -2483,14 +2483,14 @@ export class PgStore extends BasePgStore {
   async getPrincipalMempoolStxBalanceDelta(sql: PgSqlClient, principal: string): Promise<bigint> {
     const results = await sql<{ delta: string }[]>`
       WITH sent AS (
-        SELECT token_transfer_amount + fee_rate AS total
+        SELECT token_transfer_amount + COALESCE(fee_rate, 0) AS total
```

Comment:
This can end up running `null + <number>` right, does it need `COALESCE(token_transfer_amount, 0)`?

Reply:
Might be useful to add another type of tx in the tests

Reply:
Oh duh, added the coalesce to the wrong field lol... yep I will add a test for this